python/cudf/cudf/io/orc.py

```diff
@@ -434,6 +435,9 @@ def to_orc(
             "Categorical columns."
         )
 
+    if cols_as_map_type is not None and not isinstance(cols_as_map_type, list):
+        raise TypeError("cols_as_map_type can only be a list of column names")
```

Comment:
Worth adding a test that exercises this exception?

Reply:
Added a test